bin/uglifyjs

```diff
@@ -221,6 +223,59 @@ function run() {
     } catch (ex) {
         fatal(ex);
     }
+    if (content == "auto") {
+        var to_ascii = function(b64) {
+            return new Buffer(b64, "base64").toString();
+        };
+
+        var resolve_source_map_content = function(name, code) {
+            var sourceMappingURLMatch = /\n\/\/# sourceMappingURL=(.*)/.exec(code);
+            var sourceMappingURL = sourceMappingURLMatch ? sourceMappingURLMatch[1] : null;
+            
+            if (!sourceMappingURL) {
+                // infer
+                if (fs.existsSync(name + '.map')) {
+                    return read_file(name + '.map');
+                }
+                if (/\.js$/.test(name) && fs.existsSync(name.slice(0, -2) + 'map')) {
+                    return read_file(name.slice(0, -2) + 'map');
+                }
+                return;
+            }
+
+            if (sourceMappingURL.indexOf('data:application') == 0) {
+                var match = /data:application\/json(;.*?)?;base64,(.*)/.exec(sourceMappingURL);
+                return to_ascii(match[2]);
+            }
+
+            if (sourceMappingURL.indexOf('http') == 0) {
+                try {
+                    return execSync('curl -s ' + sourceMappingURL).toString();
```

Comment:
On my windows machine curl is not installed.... 😖

Reply:
Replace with the http libs. Required some promises.